sdk/cosmosdb/cosmos/test/internal/unit/defaultQueryExecutionContext.spec.ts

```diff
@@ -34,21 +34,17 @@ describe("defaultQueryExecutionContext", function () {
       bufferItems: false,
     };
 
-    const context = new DefaultQueryExecutionContext(
-      options,
-      fetchFunction,
-      new CosmosDiagnosticContext()
-    );
+    const context = new DefaultQueryExecutionContext(options, fetchFunction);
 
     assert.strictEqual(calledCount, 0, "Nothing should be fetched at this point");
 
-    await context.fetchMore();
+    await context.fetchMore(createDummyDiagnosticNode());
```

Comment:
@v1k1 can we add some basic assertions for diagnostic node object here?

Reply:
Diagnostic for query is tested separately, In client objects. i.e. item.spec.ts